Automattic/jetpack
# Extension utils: prepare for an initial release

Follow-up from #22505

#### Changes proposed in this Pull Request:

Let's add the necessary metadata so the package can be released publicly. see https://github.com/Automattic/jetpack/pull/22505#issuecomment-1024122149 for more info.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Check for typos
* Click on the Build check below.
* Click on the "Summary" section
* Click to download "jetpack-build"
* See that a new dir is included for this package.
